quickwit/quickwit-serve/src/lib.rs

```diff
@@ -511,18 +524,23 @@ pub async fn serve_quickwit(
     {
         {
             let otel_logs_index_config =
-                OtlpGrpcLogsService::index_config(&node_config.default_index_root_uri)?;
+                OtlpGrpcLogsService::index_config(&node_config.default_index_root_uri)
+                    .context("failed to load OTEL logs index config")?;
             let otel_traces_index_config =
-                OtlpGrpcTracesService::index_config(&node_config.default_index_root_uri)?;
+                OtlpGrpcTracesService::index_config(&node_config.default_index_root_uri)
+                    .context("failed to load OTEL traces index config")?;
 
-            for index_config in [otel_logs_index_config, otel_traces_index_config] {
+            for (index_name, index_config) in [
+                ("OTEL logs", otel_logs_index_config),
+                ("OTEL traces", otel_traces_index_config),
+            ] {
                 match index_manager.create_index(index_config, false).await {
                     Ok(_)
                     | Err(IndexServiceError::Metastore(MetastoreError::AlreadyExists(
                         EntityKind::Index { .. },
-                    ))) => Ok(()),
-                    Err(error) => Err(error),
-                }?;
+                    ))) => {}
+                    Err(error) => bail!("failed to create {index_name} index: {error}",),
```

Comment:
We could also just log the error here. WDYT?

Reply:
Yes... A error log might be useful here too.

Reply:
ah... I don't have a clear opinion right now.

Reply:
Ok. I'll leave it as is for now. Also, all errors in `quickwit_serve` are logged in `main`.